packages/backend-for-frontend/src/routers/swaggerRouter.ts

```diff
@@ -0,0 +1,35 @@
+import fs from "fs/promises";
+import path from "path";
+import { fileURLToPath } from "url";
+import { zodiosRouter } from "@zodios/express";
+import { bffApi } from "pagopa-interop-api-clients";
+import swaggerUi from "swagger-ui-express";
+import YAML from "yaml";
+import { config } from "../config/config.js";
+
+const BFF_API_SPEC_FILE_NAME: string = "bffApi.yml";
+const swaggerRouter = zodiosRouter(bffApi.developApi.api);
+
+if (config.bffSwaggerUiEnabled) {
+  const pkgUrl = import.meta.resolve("pagopa-interop-api-clients");
+  const pkgDir = path.dirname(fileURLToPath(pkgUrl));
+
+  const yamlPath = path.join(
+    path.dirname(pkgDir),
+    "open-api",
+    BFF_API_SPEC_FILE_NAME
```

Comment:
is it useful to let the file name be configurable if both the package name `pagopa-interop-api-clients` and folder name `open-api` are hardcoded?

Reply:
It's not an issue to leave the env var of course, but I think it's probably redundant, because it only makes sense to set it to the filename (not e.g. to set to different values in different environments). If the file is renamed in a future commit, then the value of the env var must be changed to match it: but this is equivalent to having a hardcoded value and just updating it when the file is renamed.

It could happen that we rename the file and forget to update the env var (or the hardcoded value): to prevent this, I suppose we'd need a test that checks that the file can be retrieved. Considering the low damage this issue would cause and the likelihood that the name doesn't change, I think we can do without the test.